packages/lesswrong/server/authenticationMiddlewares.ts

```diff
@@ -395,12 +396,28 @@ export const addAuthMiddlewares = (addConnectHandler: AddMiddlewareType) => {
     })
   }
 
+  // This is a mock method used in playwright tests - it gets overwritten below
+  // for actual servers
+  let profileFromAccessToken: ProfileFromAccessToken = async (token: string) => {
```

Comment:
I think this would be better as `const`, and then explicitly checking for `isE2E` everywhere we use this, to avoid accidentally using it when `hasAuth0` is false for some other reason